core/lib/workarea/ext/jbuilder/jbuilder_cache.rb

```diff
@@ -0,0 +1,31 @@
+require 'jbuilder/jbuilder_template'
+
+decorate JbuilderTemplate, with: :workarea do
+  def _cache_fragment_for(key, options)
```

Comment:
If this isn't going to use args, better to use `_cache_fragment_for(*)` so if anything about the args changes we don't have to update.